app/views/contexts/summary_context.py

```diff
@@ -14,11 +14,16 @@ def __call__(
         groups = list(self._build_all_groups(return_to))
         summary_options = self._schema.get_summary_options()
         collapsible = summary_options.get("collapsible", False)
+        headers = ["Question", "Answer given"]
+        if answers_are_editable:
+            headers.extend("Change answer")
```

Comment:
We could do some list comprehension here to reduce lines and I find it more readable but up to you.
```suggestion
        headers = ["Question", "Answer given", "Change answer"] if answers_are_editable else ["Question", "Answer given"]
```